rust-runtime/aws-smithy-types/src/timeout.rs

```diff
@@ -34,39 +34,22 @@ use std::time::Duration;
 /// # }
 /// ```
 #[derive(Clone, PartialEq, Default)]
-pub struct TimeoutConfig {
-    connect_timeout: Option<Duration>,
-    tls_negotiation_timeout: Option<Duration>,
-    read_timeout: Option<Duration>,
+pub struct SharedTimeoutConfig {
+    // connect_timeout: Option<Duration>,
+    // tls_negotiation_timeout: Option<Duration>,
+    // read_timeout: Option<Duration>,
     api_call_attempt_timeout: Option<Duration>,
     api_call_timeout: Option<Duration>,
 }
```

Comment:
so this is timeouts on operations? maybe it should be something like `ApiTimeoutConfig`?

Reply:
will there eventually be a `RootTimeoutConfig` that has both `ApiTimeoutConfig` and `HttpSettingsTimeoutConfig`?